A3A/addons/core/Stringtable.xml

```diff
@@ -2719,6 +2719,22 @@
                 <Spanish>Ganarás el juego cuando la mayoría de la población apoye a tu facción y hayas conquistado todas las bases aéreas y bases militares.</Spanish>
                 <Polish>Wygrasz grę, gdy większość populacji poprze twoją frakcję i podbijesz wszystkie lotniska i bazy wojskowe.</Polish>
             </Key>
+            <Key ID="STR_antistasi_journal_entry_text_Default_total">airportsX + milbases + outposts + resourcesX + factories + seaports;
```

Comment:
What's this at the end?
`airportsX + milbases + outposts + resourcesX + factories + seaports;`

Reply:
forgotten mistake